src/python/tests/appengine/handlers/cron/project_setup_test.py

```diff
@@ -1607,41 +1623,66 @@ def setUp(self):
         'handlers.base_handler.Handler.is_cron',
     ])
 
-    self.mock.read_data.return_value = (
-        b'{"projects": [{"build_path": '
-        b'"gs://bucket/a-b/%ENGINE%/%SANITIZER%/'
-        b'%TARGET%/([0-9]+).zip", '
-        b'"name": "//a/b", "fuzzing_engines": ["libfuzzer", "honggfuzz"], '
-        b'"sanitizers": ["address"]}]}')
+    self.mock.read_data.side_effect = _mock_read_data
 
     self.mock.ProjectConfig.return_value = mock_config.MockConfig({
-        'project_setup': {
-            'source': 'gs://bucket/projects.json',
-            'build_type': 'FUZZ_TARGET_BUILD_BUCKET_PATH',
-            'build_buckets': {
-                'afl': 'clusterfuzz-builds-afl',
-                'dataflow': 'clusterfuzz-builds-dataflow',
-                'honggfuzz': 'clusterfuzz-builds-honggfuzz',
-                'libfuzzer': 'clusterfuzz-builds',
-                'libfuzzer_i386': 'clusterfuzz-builds-i386',
-                'no_engine': 'clusterfuzz-builds-no-engine',
+        'project_setup': [
+            {
+                'source': 'gs://bucket/projects.json',
+                'build_type': 'FUZZ_TARGET_BUILD_BUCKET_PATH',
+                'build_buckets': {
+                    'afl': 'clusterfuzz-builds-afl',
+                    'dataflow': 'clusterfuzz-builds-dataflow',
+                    'honggfuzz': 'clusterfuzz-builds-honggfuzz',
+                    'libfuzzer': 'clusterfuzz-builds',
+                    'libfuzzer_i386': 'clusterfuzz-builds-i386',
+                    'no_engine': 'clusterfuzz-builds-no-engine',
+                },
+                'additional_vars': {
+                    'all': {
+                        'STRING_VAR': 'VAL',
+                        'BOOL_VAR': True,
+                        'INT_VAR': 0,
+                    },
+                    'libfuzzer': {
+                        'address': {
+                            'ASAN_VAR': 'VAL',
+                        },
+                        'memory': {
+                            'NOT_SET': 'VAL',
+                        }
+                    }
+                }
             },
-            'additional_vars': {
-                'all': {
-                    'STRING_VAR': 'VAL',
-                    'BOOL_VAR': True,
-                    'INT_VAR': 0,
+            {
+                'source': 'gs://bucket-dbg/projects.json',
+                'suffix': '_dbg',
```

Comment:
suffix feels too generic ? maybe name_suffix or config_suffix or something else ?

Reply:
Renamed to job_suffix.